src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -5387,6 +5387,74 @@ static QVariant fcnAzimuth( const QVariantList &values, const QgsExpressionConte
   }
 }
 
+static QVariant fcnBearing( const QVariantList &values, const QgsExpressionContext *context, QgsExpression *parent, const QgsExpressionNodeFunction * )
+{
+  const QgsGeometry geom1 = QgsExpressionUtils::getGeometry( values.at( 0 ), parent );
+  const QgsGeometry geom2 = QgsExpressionUtils::getGeometry( values.at( 1 ), parent );
+  QString sAuthId = QgsExpressionUtils::getStringValue( values.at( 2 ), parent );
```

Comment:
I'd avoid the 's' prefix here -- that's used to indicate static variables:

```suggestion
  QString sourceCrs = QgsExpressionUtils::getStringValue( values.at( 2 ), parent );
```